Misc/NEWS.d/next/C API/2022-04-21-17-25-22.gh-issue-91744.FgvaMi.rst

```diff
@@ -0,0 +1,10 @@
+Defining :c:macro:`Py_USING_SEMI_STABLE_API` will expose "semi-stable API",
```

Comment:
Perhaps it's worth considering to stay consistent with the spelling of "semi-stable" / "semistable". Currently the include folders are named like the latter (which I think looks good in a path), mentions in the docs like here often have the hyphen.

Even though the hyphen is grammatically slightly more correct, my personal preference would be without.

Reply:
Which also opens the question if the macro should be `Py_USING_SEMI_STABLE_API` or `Py_USING_SEMISTABLE_API`